dev-tools/README.md

```diff
@@ -92,3 +92,43 @@ Load environment variables
 ```
 audius-compose load-env discovery-provider prod
 ```
+
+# Dev Details and Adding / Updating a Service
```

Comment:
This reads a lot more densely than I would have written it, but if we can change this header to

`# Development`

and I'm ok with merge as is :)